packages/editor/src/store/utils/test/notice-builder.js

```diff
@@ -96,7 +96,7 @@ describe( 'getNotificationArgumentsForSaveFail()', () => {
 	const error = { code: '42', message: 'Something went wrong.' };
 	const post = { status: 'publish' };
 	const edits = { status: 'publish' };
-	const defaultExpectedAction = { id: SAVE_POST_NOTICE_ID };
+	const defaultExpectedAction = { id: SAVE_POST_NOTICE_ID, type: 'snackbar' };
```

Comment:
This will probably fail the tests?

Reply:
Whoops, I missed that one. Removed now. :)